configuration/backend_configuration/ampathforms/procedure-history-form.json

```diff
@@ -0,0 +1,225 @@
+                    "label": "Procedure outcome satisfactory to patient"
+                  },
+                  {
+                    "concept": "160718AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Procedure outcome satisfactory to physician"
+                  },
+                  {
+                    "concept": "160719AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Procedure outcome unsatisfactory to patient"
+                  },
+                  {
+                    "concept": "160720AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Procedure outcome unsatisfactory to physician"
+                  }
+                ]
+              },
+              "id": "procedureOutcome",
+              "label": "Procedure outcome"
+            },
+            {
+              "type": "obs",
+              "questionOptions": {
+                "rendering": "number",
+                "concept": "165929AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                "min": "1",
+                "max": "100000",
+                "step": 1
+              },
+              "id": "procedureDuration",
+              "label": "Procedure duration"
+            },
+            {
+              "type": "obs",
+              "id": "procedureDurationUnit",
+              "label": "Duration unit",
+              "questionOptions": {
+                "rendering": "select",
+                "concept": "1732AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                "answers": [
+                  {
+                    "concept": "162582AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Number of occurrences"
+                  },
+                  {
+                    "concept": "162583AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Seconds"
+                  },
+                  {
+                    "concept": "1733AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Minutes"
+                  },
+                  {
+                    "concept": "1822AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Hours"
+                  },
+                  {
+                    "concept": "1072AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Days"
+                  },
+                  {
+                    "concept": "1073AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Weeks"
+                  },
+                  {
+                    "concept": "1074AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Months"
+                  },
+                  {
+                    "concept": "1734AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Years"
+                  }
+                ]
+              },
+              "clearWhenExpression": "Number(procedureDuration) > 99 && (procedureDurationUnit === '1734AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA')",
```

Comment:
Why are other durations handled as validations, but this is a "clearWhenExpression"? It feels like we should use one tool for limits here.